plugins/source/notion/Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: test
+test:
+	go test -timeout 3m ./...
+
+.PHONY: lint
+lint:
+	@golangci-lint run --timeout 10m
+
+.PHONY: gen-docs
+gen-docs:
```

Comment:
We use the CLI to generate docs now, see https://github.com/cloudquery/cloudquery/blob/0c9bfbeda4aa4eb45843cac53301a50b931c1bf2/plugins/source/alicloud/Makefile#L34

If you use that script it should also copy the tables to the Website automatically (just replace `alicloud` with `notion` in the script)